unabridged-works-volume-1/original/07-axe-to-the-root.adoc

```diff
@@ -293,14 +293,15 @@ pleasant fruit unto life; but only sour fruit,
 finely painted and dressed for the eye and palate of death), its body, branches, leaves,
 and fruit will wither and die daily, and truth come to grow safely.
 
-=== THE AXE LAID TO THE ROOT OF THE OLD CORRUPT TREE
+=== The Axe Laid to the Root of the Old Corrupt Tree
 
+[.section-summary-preface]
 A DISTINCTION between the FAITH which is of MAN, and the FAITH which is of God:
```

Comment:
Especially in these `[.section-summary-preface]`s (which this is a perfect example of, good job!), I like to take the capitalized words and lowercase them and change them out for _italics_.  Because the section is already italicized, the italics within the italics actually render in "small-caps" which is similar to uppercase, but looks nicer.  So, anyway, can you change this line to:

```ASCIIDOC
A _Distinction_ between the _Faith_ which is of _Man,_ and the _Faith_ which is of God:
```

One other minor note: in paragraphs like this, when I switch to italics as shown above, I generally leave the first letter capitalized, because it fits with the style of the rest of the paragraph, which is why I did `_Faith_` instead of `_faith_`.